k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeUtility.java

```diff
@@ -1137,4 +1140,11 @@ public static boolean isMessage(String mimeType) {
     public static boolean isSameMimeType(String mimeType, String otherMimeType) {
         return mimeType != null && mimeType.equalsIgnoreCase(otherMimeType);
     }
+
+    static boolean isFormatFlowed(String contentType) {
+        if (TEXT_PLAIN.equalsIgnoreCase(getHeaderParameter(contentType, null))) {
```

Comment:
Extracting the results of the `getHeaderParameter()` calls to local variables should make this easier to read.